src/Reader/Extension/PodcastIndex/Feed.php

```diff
@@ -275,6 +291,195 @@ public function getPodcastIndexPeople(): array
         return $this->data['people'];
     }
 
+    /**
+     * Get the podcast trailer
+     *
+     * @return null|TrailerObject
+     */
+    public function getPodcastIndexTrailer(): object|null
+    {
+        if (array_key_exists('trailer', $this->data)) {
+            /** @var null|TrailerObject $object */
+            $object = $this->data['trailer'];
+            return $object;
```

Comment:
Psalm and PHPStorm understand this fine I think. Otherwise `@psalm-var` can be used to avoid the temporary variable:

```suggestion
            /** @var null|TrailerObject */
            return $this->data['trailer'];
```

Reply:
Then I would have to add a phpcs:ignore comment to make it pass, if that is ok?